src/main/java/com/lgtm/easymoney/payload/RequestRsp.java

```diff
@@ -0,0 +1,15 @@
+package com.lgtm.easymoney.payload;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.math.BigDecimal;
+
+@AllArgsConstructor
+@Data
+@NoArgsConstructor
+public class RequestRsp {
```

Comment:
Though this is the same as BalanceRsp, I wonder if it is better to still keep RequestRsp in order to maintain consistency in payload format and naming convention.